yb-voyager/cmd/exportData.go

```diff
@@ -168,7 +169,7 @@ func exportDataOffline() bool {
 		os.Exit(0)
 	}
 
-	if liveMigration || useDebezium {
+	if exportType != "" || useDebezium {
```

Comment:
What if `exportType` is specified as `snapshot-only`?

Define a function named `changeStreamingIsEnabled(exportType)` which would mean `s == CHANGES_ONLY || s == SNAPSHOT_AND_CHANGES` and use it in place of `liveMigration` flag.